packages/interface/src/components/ConnectButton.tsx

```diff
@@ -31,9 +30,14 @@ const ConnectedDetails = ({ openAccountModal, account, isMobile }: IConnectedDet
   );
 };
 
-export const ConnectButton = (): JSX.Element => {
-  const breakpoint = useBreakpoint();
-  const isMobile = breakpoint === "S";
+interface IConnectButtonProps {
+  showMobile: boolean;
+}
+
+export const ConnectButton = ({ showMobile }: IConnectButtonProps): JSX.Element => {
+  const isMobile = useIsMobile();
+
+  const show = useMemo(() => (showMobile && isMobile) || (!showMobile && !isMobile), [isMobile, showMobile]);
```

Comment:
It should be `isShow` because `show` is a verb which can be applicable only to functions.